app/src/main/java/com/openlattice/chronicle/UserIdentificationActivity.kt

```diff
@@ -20,35 +20,21 @@ class UserIdentificationActivity : AppCompatActivity() {
         setContentView(R.layout.activity_user_identification)
 
         settings = EnrollmentSettings(this)
-        saveBtn = findViewById(R.id.save_btn)
-        selectUserOptions = findViewById(R.id.select_user_options)
+        childUserBtn = findViewById(R.id.child_user_btn)
+        otherUserBtn = findViewById(R.id.other_user_btn)
```

Comment:
What does the button look like when it is selected?

Reply:
https://user-images.githubusercontent.com/15723547/148463797-b6157965-2fca-4ae2-9d54-c7903c5fe445.mov